setup.cfg

```diff
@@ -1,7 +1,19 @@
 [flake8]
 max-line-length=88
 exclude = docs,.venv
-ignore = E203, E501, W503, F811, F401
```

Comment:
I think `F811` used to produce false positives and why we ignored it. Not sure if the problem is fixed.

Reply:
F811 does complain when we re-define a function with the same name (which can happen when you override an entity in Bionic) but we're already using `noqa: F811` in those places, which seems reasonable to me.